spec/request/service_credential_bindings_spec.rb

```diff
@@ -63,7 +63,8 @@
   end
 
   describe 'GET /v3/service_credential_bindings/:app_guid' do
```

Comment:
This commit hasn't changed this, but should it be `binding_guid`?

Reply:
`:app_guid` is to distinguish between the other way this endpoint can be used e.g. `:key_guid`.